apps/easypid/src/use-cases/ReceivePidUseCaseCFlow.ts

```diff
@@ -1,16 +1,17 @@
-import type { MdocRecord } from '@credo-ts/core'
+import type { MdocRecord, SdJwtVcRecord } from '@credo-ts/core'
 import { pidSchemes } from '@easypid/constants'
 import {
   BiometricAuthenticationError,
   OpenId4VciAuthorizationFlow,
-  type SdJwtVcRecord,
   receiveCredentialFromOpenId4VciOffer,
   resolveOpenId4VciOffer,
+  storeCredential,
```

Comment:
shouldn't storeCredential be handled by the SDK as well, or is that a follow up task?